apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -459,6 +462,38 @@ class LivechatClass {
 			extraData,
 		});
 
+		if (isSingleContactEnabled()) {
+			let { contactId } = visitor;
+			if (contactId) {
```

Comment:
```suggestion
			if (!contactId) {
```
I'm assuming you meant to load the contactId from the visitor if it is not yet loaded (though the function is supposed to be receiving a full visitor document). 
If I misunderstood it, please add a comment on the code explaining why it is being loaded.